test/modules/backup-azure/backup_backend_test.go

```diff
@@ -34,7 +33,23 @@ import (
 	"github.com/weaviate/weaviate/usecases/config"
 )
 
-func Test_AzureBackend_Backup(t *testing.T) {
+var override = []string{"", ""}
+
+func Test_AzureBackend_Start(t *testing.T) {
+	overrides := [][]string{
+		{"", ""},
+		{"testbucketoverride", "testBucketPathOverride"},
+	}
+
+	override = overrides[0]
+	AzureBackend_Backup(t)
+	time.Sleep(5 * time.Second)
```

Comment:
do we need a 5s sleep here?

Reply:
It seems we do need it, my docker wasn't deleting the images fast enough, or something like that

Reply:
ok, are you sure that this is still the case? can you re-test once again without this sleep?

Reply:
done